apps/lockfile-explorer/src/lint.ts

```diff
@@ -0,0 +1,31 @@
+import { FileSystem, JsonFile, PackageJsonLookup } from '@rushstack/node-core-library';
+import { RushConfiguration } from '@microsoft/rush-lib';
```

Comment:
@william2958  Originally we were doing `import type` from `rush-lib` and so it was added as a devDependency.  This line here is importing an actual runtime object, so it fails when `lockfile-explorer` is globally installed (because `rush-lib` is not actually installed).  The `rush-lib` package is a very big install, so we wouldn't really want `lockfile-explorer` to depend on it.  The [rush-sdk](https://www.npmjs.com/package/@rushstack/rush-sdk) package provides a lightweight alternative.

However, since you really only need this one tiny function, (and as long as that remains true) I think a better solution is simply to copy+paste that function into this file.

Reply:
fixed